packages/tocco-util/src/validation/validation.js

```diff
@@ -0,0 +1,16 @@
+// extracts the first error message of a validation failed response object
```

Comment:
There is a common format for function comments. See:
https://en.wikipedia.org/wiki/JSDoc